Packs/CortexAttackSurfaceManagement/Scripts/GetProjectOwners/GetProjectOwners.py

```diff
@@ -0,0 +1,100 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+"""Script for identifying and recommending the most likely owners of a discovered service
+from those surfaced by Cortex ASM Enrichment.
+"""
+
+
+from typing import Dict, List, Any
+import traceback
+import re
+
+
+def is_gcp_iam_account(service_account: str) -> bool:
+    """
+    Determine whether service account is user-managed
+
+    Docs: https://cloud.google.com/iam/docs/service-account-types
+    """
+    return bool(re.search("^.+@[^\.]+(\.iam\.gserviceaccount\.com)$", service_account))
+
+
+def extract_project_name(service_account: str) -> str:
+    """
+    Extract project name from GCP IAM service account
+    """
+    match = re.search(r"(?<=@)[^\.]+(?=\.iam\.gserviceaccount\.com)", service_account)
+    if match:
+        return match.group()
+    else:
+        raise ValueError(f"Could not extract project name from service account {service_account}")
+
+
+def get_iam_policy(project_name: str) -> List[Dict[str, Any]]:
+    """
+    Retrieve IAM policy for project
+    """
+    try:
+        return demisto.executeCommand("gcp-iam-project-iam-policy-get", args={"project_name": f"projects/{project_name}"})
+    except Exception as ex:
+        raise RuntimeError(f"Error retrieving IAM policy for GCP project {project_name}") from ex
```

Comment:
we will have to treat this differently because it could cause playbooks to fail where a user doesn't have GCP-IAM integration setup.  We should check for integration i playbook or add some handling in here and maybe just return to war room some like like "Error retrieving IAM policy for GCP project {project_name}" instead of Exception.